src/Stratis.Bitcoin.Features.Consensus/Rules/PowConsensusRuleEngine.cs

```diff
@@ -72,14 +72,17 @@ public override void Initialize(ChainedHeader chainTip)
 
             HashHeightPair coinViewTip = coinDatabase.GetTipHash();
 
+            this.logger.LogInformation("Rewinding coin view from '{0}' to '{1}'.", coinViewTip, chainTip);
```

Comment:
This rewind log message isn't always applicable ;)

Reply:
Fixed.